apps/ios/Shared/Views/Chat/ChatItem/FramedItemView.swift

```diff
@@ -40,19 +40,20 @@ struct FramedItemView: View {
     var body: some View {
         let v = ZStack(alignment: .bottomTrailing) {
             VStack(alignment: .leading, spacing: 0) {
+                let headerPadBottom = chatItem.quotedItem == nil && chatItem.meta.itemForwarded == nil
```

Comment:
this is a bit of a hack, you are mixing two concerns and have parallel ifs here - better make another function for cases where padding is needed (in addition to framedItemHeader)

Reply:
although maybe I don't get the problem...